lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.hvac_systems.rb

```diff
@@ -0,0 +1,245 @@
+class ASHRAE901PRM < Standard
+  DESIGN_CHILLED_WATER_TEMPERATURE = 44 # Loop design chilled water temperature (F)
+  DESIGN_CHILLED_WATER_TEMPERATURE_DELTA = 10.1 # Loop design chilled water temperature  (deltaF)
+  CHW_OUTDOOR_TEMPERATURE_HIGH = 80 # Chilled water temperature reset at high outdoor air temperature (F)
+  CHW_OUTDOOR_TEMPERATURE_LOW = 60 # Chilled water temperature reset at low outdoor air temperature (F)
+  CHW_OUTDOOR_HIGH_SETPOINT = 44 # Chilled water setpoint temperature at high outdoor air temperature (F)
+  CHW_OUTDOOR_LOW_SETPOINT = 54 # Chilled water setpoint temperature at low outdoor air temperature (F)
+  CHILLER_CHW_LOW_TEMP_LIMIT = 36 # Chiller leaving chilled water lower temperature limit (F)
+  CHILLER_CHW_COND_TEMP = 95 # Chiller entering condenser fluid temperature (F)
+  PRIMARY_PUMP_POWER = 9 # primary pump power 15 W/gpm
+
+  # Creates a chilled water loop and adds it to the model.
+  # This function creates a primary and secondary loop configuration of the cooling loop
+  #
+  # @param model [OpenStudio::Model::Model] OpenStudio model object
+  # @param system_name [String] the name of the system, or nil in which case it will be defaulted
+  # @param cooling_fuel [String] cooling fuel. Valid choices are: Electricity, DistrictCooling
+  # @param dsgn_sup_wtr_temp [Double] design supply water temperature in degrees Fahrenheit, default 44F
+  # @param dsgn_sup_wtr_temp_delt [Double] design supply-return water temperature difference in degrees Rankine, default 10R
+  # @param chw_pumping_type [String] valid choices are const_pri, const_pri_var_sec
+  # @param chiller_cooling_type [String] valid choices are AirCooled, WaterCooled
+  # @param chiller_condenser_type [String] valid choices are WithCondenser, WithoutCondenser, nil
+  # @param chiller_compressor_type [String] valid choices are Centrifugal, Reciprocating, Rotary Screw, Scroll, nil
+  # @param num_chillers [Integer] the number of chillers
+  # @param condenser_water_loop [OpenStudio::Model::PlantLoop] optional condenser water loop for water-cooled chillers.
+  #   If this is not passed in, the chillers will be air cooled.
+  # @param waterside_economizer [String] Options are 'none', 'integrated', 'non-integrated'.
+  #   If 'integrated' will add a heat exchanger to the supply inlet of the chilled water loop
+  #     to provide waterside economizing whenever wet bulb temperatures allow
+  #   If 'non-integrated' will add a heat exchanger in parallel with the chiller that will operate
+  #     only when it can meet cooling demand exclusively with the waterside economizing.
+  # @return [OpenStudio::Model::PlantLoop] the resulting chilled water loop
+  def model_add_chw_loop(model,
```

Comment:
why do you need duplicate this constructor method?  can you edit the original method to accommodate it, or otherwise modify the loop it creates instead of duplicating the method?

I hope to move hvac_systems.rb and its methods to generic model generation code that doesn't require building a Standard to access.  If that happens, this will no longer overload that method.

Reply:
Added new method called plant_loop_set_chw_pri_sec_configuration to incorporate the alternate configuration with a heat exchanger between the primary and secondary loop that was implemented for the stable baseline. This allows the Prototype.hvac_systems.rb version to handle the stable baseline case.